build-system/tasks/e2e/index.js

```diff
@@ -40,12 +40,12 @@ function installPackages_() {
 }
 
 function buildRuntime_() {
-  execOrDie('gulp build');
+  execOrDie('gulp dist');
```

Comment:
This should be `gulp dist --fortesting`, so that the runtime can be served from `localhost:8000`